packages/esbuild-plugin/__tests__/__fixtures__/.babelrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "ie": 11
```

Comment:
Please use the same config as the other plugins.

Reply:
removed this fixture `babelrc` altogether, since its not used